### DISCUSSION TITLE
Contributing translation to updated handle

### DISCUSSION THREAD

**Discussion by Pedroobraga** [ricktu288/ray-optics#169] [General]
Hello. I have recently noticed that the completed % count for the brazilian portuguese translation has dropped. Looking around the simulator, the only new text I have noticed is the handle update. On the main translation files in simulator/locales, however, there seems to be no change. Is there anywhere else I should be looking for this new text or should I just wait until it is added there?

**Comment by ricktu288** [comment]
The translation files are now moved to /locales/ID.json

https://github.com/ricktu288/ray-optics/blob/master/locales/pt-BR.json

See the items with incomplete=true

**Reply by Pedroobraga** [reply]
Thank you. It seems I had just forgotten to update my fork
